cpp/src/io/parquet/experimental/hybrid_scan_helpers.cpp

```diff
@@ -40,17 +40,32 @@ using io::detail::inline_column_buffer;
 using parquet::detail::CompactProtocolReader;
 using parquet::detail::equality_literals_collector;
 using parquet::detail::input_column_info;
-using parquet::detail::row_group_info;
+using text::byte_range_info;
 
 namespace {
-
+// Construct a vector of all row group indices from the input vectors
 [[nodiscard]] auto all_row_group_indices(
   host_span<std::vector<cudf::size_type> const> row_group_indices)
 {
   return std::vector<std::vector<cudf::size_type>>(row_group_indices.begin(),
                                                    row_group_indices.end());
 }
 
+// Compute total number of input row groups
+[[nodiscard]] cudf::size_type compute_total_row_groups(
```

Comment:
is this the same as https://github.com/rapidsai/cudf/pull/18716?

Reply:
Oh no, this computes the total (accumulate) number of row groups in the input span of row group indices.